src/api/main.py

```diff
@@ -1,8 +1,12 @@
 from fastapi import FastAPI
 from masquer.__about__ import __version__
+from src.logging_config import setup_logging, get_logger
```

Comment:
Imports need updating - the FastAPI app errors out from the Docker container due to not recognising the `src` directory